tests/jballerina-unit-test/src/test/resources/test-src/record/readonly_record_fields.bal

```diff
@@ -924,6 +924,61 @@ function assertTrue(any|error actual) {
     assertEquality(true, actual);
 }
 
+type RecordWithNeverField record {|
```

Comment:
Shall we add these before the `const ASSERTION_ERROR_REASON = "AssertionError";` declaration? Let's keep the assertion helper functions at the end of the file as before.